dev/bots/test.dart

```diff
@@ -232,239 +239,169 @@ Future<void> _runToolTests() async {
   await selectSubshard(subshards);
 }
 
-/// Verifies that AOT, APK, and IPA (if on macOS) builds the
-/// examples apps without crashing. It does not actually
-/// launch the apps. That happens later in the devicelab. This is
-/// just a smoke-test. In particular, this will verify we can build
-/// when there are spaces in the path name for the Flutter SDK and
+/// Verifies that AOT, APK, and IPA (if on macOS) builds the examples apps
+/// without crashing. It does not actually launch the apps. That happens later
+/// in the devicelab. This is just a smoke-test. In particular, this will verify
+/// we can build when there are spaces in the path name for the Flutter SDK and
 /// target app.
 Future<void> _runBuildTests() async {
   final Stream<FileSystemEntity> exampleDirectories = Directory(path.join(flutterRoot, 'examples')).list();
   await for (FileSystemEntity fileEntity in exampleDirectories) {
-    if (fileEntity is! Directory) {
+    if (fileEntity is! Directory)
```

Comment:
I know we disagree on this, but this makes my job as a reviewer harder. Someone who later comes in and adds a line before or after continue that's indendented will easily miss that there are no brackets here, and we don't use dartfmt to enforce that indendtation ends up where it belongs (and I believe dartfmt would potentially put this on the same line anyway, which we don't want). 

This makes code review harder, and it's something that even very experienced code reviewers has missed on our team.

Reply:
I don't understand how you can miss that there's no braces here.

```dart
  if (fileEntity is! Directory)
    continue;
    print('Hello');
```

...screams "broken indentation" to me. Maybe it's because I never really wrote much Python?

Reply:
(it also screams "dead code" to me)

Reply:
(i've added braces to the two if statements in this function)